src/CommunityToolkit.Maui/Converters/IsNullOrEmptyConverter.shared.cs

```diff
@@ -32,5 +32,5 @@ public class IsNullOrEmptyConverter : ValueConverterExtension, ICommunityToolkit
 		=> throw new NotImplementedException();
 
 	internal static bool ConvertInternal(object? value) =>
-		value == null || (value is string str && string.IsNullOrWhiteSpace(str));
-}
\ No newline at end of file
+		value == null || (value is string str && string.IsNullOrEmpty(str));
```

Comment:
It looks like the `is` vs `==` checking in `.editorconfig` isn't being forceful here. I can check on the severity in a new PR